crowbar_framework/app/models/neutron_service.rb

```diff
@@ -276,7 +276,7 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
     allocate_virtual_ips_for_any_cluster_in_networks_and_sync_dns(server_elements, vip_networks)
 
     network_nodes.each do |n|
-      net_svc.allocate_ip "default", "public", "host",n
+      net_svc.enable_interface "default", "nova_floating", n
```

Comment:
We probably need something similar in the nova barclamp for the DVR case, I think.